### PR TITLE
Adjust tox test matrix to include Wagtail 5.1

### PR DESCRIPTION
**Pull request by nickmoreton**
# Wagtail 5.1 update

Support ticket: https://torchbox.monday.com/boards/1124794299/pulses/1124795123

There are no changes to the working code, just changes to add 5.1 to the test matrix.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{38,39,310}-dj{32,41}-wagtail{41,42}
-    py{310,311}-dj{41}-wagtail{41}
-    py{310,311}-dj{42}-wagtail{50,main}
+    py{38,39,310}-dj{32}-wagtail{41,42,50,51}
+    py{38,39,310,311}-dj{41}-wagtail{41,42}
+    py{310,311}-dj{42}-wagtail{50,51,main}
```
Comment (katdom13) [inline tox.ini]: While I think the above is OK, I think we should standardize the envlist across these sites.
For instance, the above doesn't have tests for py{x]-dj41-wagtail{50,51}

```suggestion
    py{38,39,310}-dj{32,41}-wagtail{41,42,50,51}
    py311-dj41-wagtail{41,42,50,51}
    py311-dj42-wagtail{50,51}
```

**Comment by katdom13** [comment]
Should we be running `wagtail updatemodulepaths` and be cleaning up code from Wagtail versions older than 4.1?
Edit: Nevermind, it's all clean. Must have been looking at another directory.

**Review by katdom13 — CHANGES_REQUESTED** [review]

**Review by katdom13 — APPROVED** [review]
LGTM, Just have an optional suggestion

File: tox.ini (hunk 1 above)
Comment (nickmoreton) [reply tox.ini]: Hi @katdom13 Thank you this is a good idea.

**Pull request merged**